Entities/MOSRotating.cpp

```diff
@@ -454,7 +454,8 @@ int MOSRotating::ReadProperty(std::string propName, Reader &reader)
     {
         AEmitter *pEmitter = new AEmitter;
         reader >> pEmitter;
-        m_Emitters.push_back(pEmitter);
```

Comment:
It looks like this used to allow you to add wounds directly via ini but you didn't add support for an AddWound property. Am I mistaken, or is this missing? If so, I guess add it please (though I don't know how often it'd ever be used).

Reply:
There was never an option to add wounds _specifically_ via ini.
This was pushing emitters using AddEmitter to m_Emitters, which no longer exists and emitters are being pushed into m_Attachables instead.

Reply:
Yeah, and pushing into m_Emitters was adding wounds, cause m_Emitters stored all wounds.

Reply:
Okay, resolved. We'll remove the janky wound adding via ini since no one wanted it anyway, and mention it in the changelog.